src/common/types/blob.cpp

```diff
@@ -63,6 +63,20 @@ string Blob::ToString(string_t blob) {
 	return string(buffer.get(), str_len);
 }
 
+void Blob::ToStringAsIs(string_t blob, char *output) {
```

Comment:
We also won't need these two functions anymore then, right?

Reply:
I reverted this change.